examples/python/phi3v.py

```diff
@@ -29,9 +28,10 @@ def run(args: argparse.Namespace):
                 "Image Path (comma separated; leave empty if no image): "
             ).split(",")
         ]
+        image_paths = [image_path for image_path in image_paths if len(image_path)]
```

Comment:
Why is this needed?

Reply:
The image paths are otherwise passed as empty strings to the processor and the following error occurs.

```
Loading model...
Image Path (comma separated; leave empty if no image): 
['']
Loading images...
Traceback (most recent call last):
  File "/path/to/onnxruntime-genai/examples/python/phi3v.py", line 79, in <module>
    run(args)
  File "/path/to/onnxruntime-genai/examples/python/phi3v.py", line 42, in run
    raise FileNotFoundError(f"Image file not found: {image_path}")
FileNotFoundError: Image file not found:
```

Reply:
I see. This would happen if we provide empty spaces in the file paths without listing the files. Cool, thanks.